### PR TITLE
Ops/controller dockerfile

### PR DESCRIPTION
**Pull request by Courtcircuits**
Dockerfile for controller deployment

### CODE REVIEW

File: controller/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+COPY controller/.sqlx .sqlx
+RUN apt update && apt upgrade -y
+RUN apt install -y protobuf-compiler libprotobuf-dev
+RUN protoc --version
+RUN --mount=type=cache,target=/opt/target/ \
+    --mount=type=bind,source=controller/Cargo.toml,target=Cargo.toml  \
+    --mount=type=bind,source=controller/Cargo.lock,target=Cargo.lock  \
+    --mount=type=cache,target=/usr/local/cargo/registry/ \
+    cargo build --release --bin controller && \
+    cp ./target/release/controller /bin/server
```
Comment (Nayrode) [inline controller/Dockerfile:29]: why calling it "server"?

File: controller/src/logs/log_repository.rs [hunk 2]
```diff
@@ -6,7 +6,9 @@ use super::Log;
 
 #[derive(Debug, Clone)]
 pub struct LogDTO {
+    #[allow(dead_code)]
```
Comment (Nayrode) [inline controller/src/logs/log_repository.rs:9]: Will it be used in the future ?

**Review by Nayrode — CHANGES_REQUESTED** [review]

File: controller/Dockerfile (hunk 1 above)
Comment (Courtcircuits) [reply controller/Dockerfile:29]: The binary name is not a big deal

File: controller/src/logs/log_repository.rs (hunk 2 above)
Comment (Courtcircuits) [reply controller/src/logs/log_repository.rs:9]: Yep

**Pull request merged**